boilerplate/registry/index.html

```diff
@@ -0,0 +1,288 @@
+      <p>The <a>registry table</a> MUST provide a link to the version control system against
+        which any issues and pull requests can be opened.</p>
+      <section id="registry-entries">
+        <h4>Registry Entries</h4>
+        <p>Each <a>registry entry</a> has a <a>status</a>, a unique <dfn>key</dfn>, and
+          if appropriate, other <dfn>fields</dfn>, for example any notes,
+          a description, or a reference to some other
+          defining entity.
+        </p>
+        <p>The <a>registry table</a> definition MUST define
+          the <a>fields</a> and the <a>key</a> to be used
+          in each <a>registry entry</a>.</p>
+
+        <section id="registry-entry-status">
+          <h5>Status</h5>
+          <p>The <a>registry entry</a> <dfn>status</dfn> field reflects the maturity of that entry.
+            Permitted values are:</p>
+          <pre>
+            Provisional
+            Final
+            Deprecated
+          </pre>
+          <p>No other values are permitted.</p>
+          <section id="provisional-entry">
+            <h6>Provisional</h6>
+            <p><a>Registry entries</a> with a <a>status</a> of <dfn><code>Provisional</code></dfn> MAY be changed or deleted.
+              Their <a>status</a> may be changed to <a><code>Final</code></a> or <a><code>Deprecated</code></a>.</p>
+            <p><a>Registry entry</a> <a>keys</a> in <a><code>Provisional</code></a> entries
+              that were later deleted MAY be reused.</p>
+            <p>Newly created <a>registry entries</a> SHOULD have <a>status</a> <a><code>Provisional</code></a>.</p>
+          </section>
+          <section id="final-entry">
+            <h6>Final</h6>
+            <p><a>Registry entries</a> with a <a>status</a> of <dfn><code>Final</code></dfn> MUST NOT be deleted or changed.
+              Their <a>status</a> MAY be changed to <a><code>Deprecated</code></a>.</p>
+            <p><a>Registry entry</a> <a>keys</a> in <a><code>Final</code></a> entries MUST NOT be reused.</p>
+            <p>Newly created <a>registry entries</a> MAY have <a>status</a> <a><code>Final</code></a>.</p>
+          </section>
+          <section id="deprecated-entry">
+            <h6>Deprecated</h6>
+            <p><a>Registry entries</a> with a <a>status</a> of <dfn><code>Deprecated</code></dfn> MUST NOT be deleted or changed.
+              Their <a>status</a> MAY be changed to <a><code>Final</code></a>
+              unless that would result in a duplicate <a>key</a> within the set of entries whose
+              <a>status</a> is either <a><code>Provisional</code></a> or <a><code>Final</code></a>.</p>
+            <p><a>Registry entry</a> <a>keys</a> in <a><code>Deprecated</code></a> entries
+              that were previously <code>Provisional</code> and never <a><code>Final</code></a>
+              MAY be reused.</p>
+            <p><a>Registry entry</a> <a>keys</a> in <a><code>Provisional</code></a> entries
```

Comment:
This `Provisional` seems better to be replaced with `Deprecated` for me,,, no?? (previous: Final and current: Provisional seems to be prohibited)

Reply:
Yes that's a bug, thanks, will fix.

Reply:
Done in 05ef9ab. Good catch!